src/app/components/ui-card/ui-card.component.ts

```diff
@@ -8,4 +9,7 @@ import { GridMediaItem } from 'src/app/types/ui-types/grid-media-item';
 })
 export class UiCardComponent {
   @Input() public dataItem?: GridMediaItem;
+  @Input() public imgSize = 'w185';
```

Comment:
Do I right remember, that you have enum with these values? Probably better to use it here

Reply:
I don't have enum, im just guarding image size in imgSrc pipe to be sure that i'm not trying to request image size which isnt available

Reply:
Fixed, moved string to enum.